docs/src/pages/components/tabs/UnstyledTabCustomized.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { styled } from '@mui/system';
+import Tabs from '@mui/base/TabsUnstyled';
+import TabsListUnstyled from '@mui/base/TabsListUnstyled';
```

Comment:
Just a thought I had - how would you feel about placing all tab-related components under @mui/base/TabsUnstyled and have just "top-level" components directly under @mui/base (by that I mean components that could be used on their own, which isn't the case for TabPanel, among others)

Reply:
Interesting. I tried to follow what we have so far, which is components to be defined at level 3 of the imports unless they are private (in that case they are in their's parent folder). I think we can discuss this separately to avoid making a decision hidden in an unrelated PR comment. Would that work?

Reply:
Yes, absolutely, I didn't mean to change it in this PR. It just came to my mind while looking at the code.